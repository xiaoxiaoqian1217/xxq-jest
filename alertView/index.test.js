import React from 'react';
import Test1 from './index'
import Adapter from 'enzyme-adapter-react-16';
import Enzyme, { shallow } from 'enzyme';
import { expect } from 'chai';

Enzyme.configure({ adapter: new Adapter() });

// it('renders correctly', () => {
//   const tree = renderer
//   .create(<Test1  />)
//           .toJSON();

//   expect(tree).toMatchSnapshot();
// });
it('allows us to set props', () => {
  const wrapper = shallow((<Test1 baz="11" />));
  expect(wrapper.prop('baz')).to.equal('11');

}); 