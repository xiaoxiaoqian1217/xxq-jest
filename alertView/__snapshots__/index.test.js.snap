// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="shadowView"
  id="alertView"
>
  <div
    className="alertView"
  >
    <div
      className="titleView"
    />
    
    <div
      className="contentView"
    />
    <div
      className="buttonsView"
    >
      <div
        className="buttonItem"
      >
        取消
      </div>
      <div
        className="buttonItem"
      >
        复制
      </div>
    </div>
  </div>
</div>
`;
